docs/tutorials/buttons.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Using the Buttons
+subject: Buttons
+---
+
+* Table of Contents
+{:toc}
+
+![EV3 Buttons](/images/lego_ev3_buttons.jpg)
```

Comment:
I can give you a high res image for the EV3 I use for my video tutorials - it's an LDraw rendering - just ask